openedx/features/partners/give2asia/views.py

```diff
@@ -164,6 +165,15 @@ def create_account_with_params_custom(request, params, partner):
     # Perform operations that are non-critical parts of account creation
     preferences_api.set_user_preference(user, LANGUAGE_KEY, get_language())
 
+    try:
+        partner_communities = PartnerCommunity.objects.filter(partner=partner)
```

Comment:
We can make it more simple
`partner_communities = partner.communities.all()`
or 
`partner_communities = partner.partner_communities.all()`

use whatever the related_name is

Reply:
Done